test/jit/test_backends.py

```diff
@@ -492,8 +497,12 @@ class JitBackendTestCaseWithCompiler(JitTestCase):
 
     def setUp(self):
         super().setUp()
-        torch_root = Path(__file__).resolve().parent.parent.parent
-        p = torch_root / 'build' / 'lib' / 'libbackend_with_compiler.so'
+        if IS_IN_CI:
```

Comment:
Are these updates no longer the sitepackages path?

Reply:
> Are these updates no longer the sitepackages path?

`Path(torch.__file__).resolve().parent` is equivalent to `site.getsitepackages()[0] + '/torch'` ( = `...envs/test/lib/python3.9/site-packages/torch`). Since adding `IS_IN_CI` here, I think it's better to use the same way to get the relevant paths

Reply:
`Path(torch.__file__).resolve().parent` is equivalent to `site.getsitepackages()[0] + '/torch'` ( = `...envs/test/lib/python3.9/site-packages/torch`). Since adding `IS_IN_CI` here, I think it's better to use the same way to get the relevant paths